suite-common/formatters/src/formatters/prepareYearMonthFormatter.ts

```diff
@@ -0,0 +1,11 @@
+import { format } from 'date-fns';
+import enUS from 'date-fns/locale/en-US';
+
+import { makeFormatter } from '../makeFormatter';
+
+export const YearMonthFormatter = makeFormatter<Date, string>(
```

Comment:
I know this is copy & paste & tweak, but I would suggest insert locale from outside or rename this to something like EnYearMonthFormatter and move it to `/suite-native`. Wdyt?

Reply:
I would also suggest to delete `MonthNameFormatter`

Reply:
Removing `MonthNameFormatter` is probably doable if it's not used elsewhere. Not sure about the rest since `locale` is hardcoded into other formatters as well. I don't know what the motivation behind `FormatterProviderContext` was and whether it makes sense to have the formatter there. 🤷‍♂️

Reply:
We could also just change the format in `MonthNameFormatter` and leave the rest as is. Sounds as the best option at the moment TBH.